src/lib/storage/StorageFileApi.ts

```diff
@@ -226,13 +226,17 @@ export class StorageFileApi {
    */
   async list(
     path?: string,
-    options?: SearchOptions
+    options?: SearchOptions,
+    parameters?: FetchParameters
```

Comment:
Can you add  `parameters` description into method comment. It will be used to generate docs on supabase website

Reply:
oooh yes gotcha! will do!

Reply:
okay just updated!